### ISSUE TITLE
Loading an Arrow or Parquet file should preload apache-arrow/parquet-wasm

### ISSUE THREAD

**Issue by mbostock** [observablehq/framework#763]
You can workaround this manually with:

```js
import "npm:apache-arrow";
import "npm:parquet-wasm/esm/arrow1.js";
```

We’ll only be smart enough to do this in simple cases of `FileAttachment(name).parquet()` though.

**Comment by mbostock** [comment]
Same is true for CSV: loading a `.csv` file should preload `npm:d3-dsv`.

**Comment by mbostock** [comment]
Fixed in #843.
